workspaces/ui-v2/src/store/shapes/slice.ts

```diff
@@ -30,11 +30,11 @@ const shapesSlice = createSlice({
         loading: false,
         data: action.meta.arg.rootShapeId,
       };
-      const { shapeMap, fieldIdToShapeId } = action.payload;
+      const { shapeMap, fieldMap } = action.payload;
       for (const [shapeId, reduxShapes] of Object.entries(shapeMap)) {
         state.shapeMap[shapeId] = reduxShapes;
       }
-      state.fieldIdToShapeId = fieldIdToShapeId;
+      state.fieldMap = fieldMap;
```

Comment:
@niclim update so that this doesn't get overridden since we call this in multiple steps

Reply:
thanks for the changes